### ISSUE TITLE
Problem GNOME - trusty - Crouton

### ISSUE THREAD

**Issue by rayolo** [dnschneid/crouton#2763]
Hello, I cant run startgnome cause I have a problem. Could you help me?

Loading extension GLX
crouton: version 1-20160902144033~master:9f8e9a22
release: trusty
architecture: armhf
xmethod: xorg
targets: core,audio,touch,xorg,x11,gtk-extra,extension,gnome,keyboard
host: version 8743.4.0 (Official Build) dev-channel veyron_minnie 
kernel: Linux localhost 3.14.0 #1 SMP PREEMPT Mon Aug 29 23:26:02 PDT 2016 armv7l armv7l armv7l GNU/Linux
freon: yes
No Chromium OS X server is available.
Running exit commands...
/usr/bin/xinit: connection to X server lost

waiting for X server to shut down Hangup
Hangup
Hangup
(EE) Server terminated successfully (0). Closing log file.

**Comment by DennisLfromGA** [comment]
Try adding the 'xiwi' target to your chroot on your ASUS Chromebook Flip C100, like so:

`sudo sh ~/Downloads/crouton/crouton -n trusty -t xiwi -u`

Then try your desktop again: `sudo startgnome`

**Comment by rayolo** [comment]
the first command is right? /crouton/crouton ??

**Comment by rayolo** [comment]
chronos@localhost ~ $ sudo sh ~/Downloads/crouton/crouton -n trusty -t xiwi -u
sh: Can't open /home/chronos/user/Downloads/crouton/crouton

**Comment by rayolo** [comment]
**now is worse than before:**
Loading extension GLX
/usr/bin/xinit: XFree86_VT property unexpectedly has 0 items instead of 1
crouton: version 1-20160902144033~master:9f8e9a22
release: trusty
architecture: armhf
xmethod: xiwi
targets: xiwi,core,audio,touch,xorg,x11,gtk-extra,extension,gnome,keyboard
host: version 8743.4.0 (Official Build) dev-channel veyron_minnie 
kernel: Linux localhost 3.14.0 #1 SMP PREEMPT Mon Aug 29 23:26:02 PDT 2016 armv7l armv7l armv7l GNU/Linux
freon: yes
Error: not connected.
Cannot connect to extension, retrying...
Error: not connected.
Cannot connect to extension, retrying...
Error: not connected.
Cannot connect to extension, retrying...
Error: not connected.
Cannot connect to extension, retrying...
Error: not connected.
Cannot connect to extension, retrying...
Error: not connected.
Cannot connect to extension, retrying...
Error: not connected.
Cannot connect to extension, retrying...
Error: not connected.
Cannot connect to extension, retrying...
Error: not connected.
Cannot connect to extension, retrying...
Error: not connected.
Cannot connect to extension, retrying...
Unable to start display, make sure the crouton extension is installed
and enabled, and up to date. (download from http://goo.gl/OVQOEt)
Running exit commands...
/usr/bin/xinit: connection to X server lost

waiting for X server to shut down Hangup
Hangup
Hangup
(EE) Server terminated successfully (0). Closing log file.

Unmounting /media/removable/rayoloSD/chroots/trusty...
Sending SIGTERM to processes under /media/removable/rayoloSD/chroots/trusty

**Comment by rayolo** [comment]
I downloaded the crouton extension here (download from http://goo.gl/OVQOEt) and still doesnt work

**Comment by rayolo** [comment]
nobody can help me ???

**Comment by DennisLfromGA** [comment]
@rayolo,

See [this thread](https://github.com/dnschneid/crouton/issues/2464#issuecomment-188301652) where a developer discusses gnome on the Flip.
He suggests using xfce4 or lxde instead.

Hope this heops,
-**DennisL**

**Comment by ArielleA** [comment]
Hi, @rayolo,

I have been playing with this for the last couple of days. Actually I am addicted to Evolution so I don't really care what Desktop environment I use (I happen to like Mate) but in getting Evolution to work in armhf on my CB5-311 I had to find a solution around not using softrast but instead getting LLVMpipe. Evolution was complaining about not getting a eGL surface when starting from cli. So knowing that stretch has llvmpipe built in (I used it in my native install on an Asus C201) tried it and it resolves this error. I have a working Gnome session, and the apps work too.

Hope this is useful and you're not too tied into trusty, I didn't try Xenial as I am not sure it has mate built in.

Hope this helps

Arielle :)
